Description: Enable release mode
Author: Alexander Wolf <alex.v.wolf@gmail.com>
Index: stellarium/CMakeLists.txt
===================================================================
--- stellarium.orig/CMakeLists.txt	2014-03-16 20:33:10.906530070 +0700
+++ stellarium/CMakeLists.txt	2014-03-16 20:33:34.389970990 +0700
@@ -19,7 +19,7 @@
 
 ########### Main global variables ###########
 IF(NOT CMAKE_BUILD_TYPE)
-  SET(CMAKE_BUILD_TYPE Debug CACHE STRING "Choose the type of build, options are: None Debug GProf Valgrind Release RelWithDebInfo MinSizeRel." FORCE)
+  SET(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build, options are: None Debug GProf Valgrind Release RelWithDebInfo MinSizeRel." FORCE)
 ENDIF()
 
 # Add gprof build options if necessary.  Note gmon.out will be created in working directory when Stellarium is executed
